### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by ccoVeille**
- **chore: fix typo in code**
- **chore: fix tool names or registered trademarks**
- **documentation: fix Markdown fenced block identifier**

### CODE REVIEW

**Comment by ccoVeille** [comment]
I was unsure if this one was a typo or not

https://github.com/cosmtrek/air/blob/e3b6eafe8afd4ac8640421bad573ea5e655592cd/runner/util.go#L361C6-L361C17

What could mean the setTage2Map ?

File: README-zh_cn.md [hunk 1]
```diff
@@ -28,17 +28,21 @@ Air 是为 Go 应用开发设计的另外一个热重载的命令行工具。只
...
+```shell
+air --build.cmd "go build -o bin/api cmd/run.go" --build.bin "./bin/api" --build.exclude_dir "templates,build"
+```
 
 ## 安装
 
 ### 推荐使用 install.sh
 
-```bash
+```shell
```
Comment (xiantang) [inline README-zh_cn.md:45]: what the different betweent shell and bash

**Comment by xiantang** [comment]
> I was unsure if this one was a typo or not
> 
> https://github.com/cosmtrek/air/blob/e3b6eafe8afd4ac8640421bad573ea5e655592cd/runner/util.go#L361C6-L361C17
> 
> What could mean the setTage2Map ?
should be setTag2Map

**Comment by xiantang** [comment]
well done!

File: README-zh_cn.md (hunk 1 above)
Comment (ccoVeille) [reply README-zh_cn.md:45]: In term of highlighting none

Shell is the reference and others are aliases


https://github.com/github-linguist/linguist/blob/e2012cd67867e6287ae5ee96c8fe0b92f9a88efa/lib/linguist/languages.yml#L6530-L6537

Comment (ccoVeille) [reply README-zh_cn.md:45]: @xiantang as I'm unsure you saw my reply

**Pull request merged**